sycl/source/CMakeLists.txt

```diff
@@ -222,6 +222,18 @@ function(add_sycl_rt_library LIB_NAME LIB_OBJ_NAME)
     target_compile_options(${LIB_OBJ_NAME} PRIVATE
       -Winstantiation-after-specialization)
   endif()
+
+  # When building using icx on Windows, the VERSION file
+  # produced by cmake is used in source code
+  # when including '<version>' because Windows is
+  # case-insensitive and icx adds the build directory
+  # to the system header search path.
```

Comment:
Wow... Just speechless...

Do you know if "VERSION" is some standard naming? If not, maybe we can rename it instead of this workaround.

Reply:
the variable as as CMake varibale is standard but writing it to disk seems to be somewhat standard but not done by every project. im pretty sure here it's coming from [this](https://github.com/oneapi-src/level-zero/blob/master/CMakeLists.txt#L77) code.